### DISCUSSION TITLE
Add Categories to Invoices

### DISCUSSION THREAD

**Discussion by heredia21** [btcpayserver/btcpayserver#5492] [Ideas / Feature Requests]
To better organize invoices, it would be useful if categories can be added to invoices.

**Comment by pavlenex** [comment]
Can you give an example? Category based on what criteria? There are status filters and date, this is not enough?

**Comment by heredia21** [comment]
Sure. Let's say I want to start collecting rent as a landlord. I want the ability to add my own custom categories. Ex:

Maintenance
Utilities
Repairs
Rent

Then be able to run reports off those custom categories.

**Reply by dstrukt** [reply]
Adding a label or a category for an invoice, similar to transactions, makes sense!

I'm guessing you would you like to be able to label/tag/categorize (whatever language we go with) before you send it out (e.g. at the Create Invoice step) and afterwards in the Invoice List view (to potentially categorize older or missed invoices), yes? Wanted to clarify expectations a bit more clearly.

**Reply by heredia21** [reply]
Yes, that is correct. Add category at Create invoice step then when you go to Reporting - Payments it exports the CSV with a Category column that will either be null or shown the Category you added next to the payment.
